cloudferrylib/scheduler/scheduler.py

```diff
@@ -41,15 +41,18 @@ def __init__(self, namespace=None, cursor=None):
         self.map_func_task[BaseTask()] = self.task_run
 
     def event_start_task(self, task):
+        LOG.info('%s Start task: %s', '-' * 8, str(task).split("|")[1])
```

Comment:
`str(task).split("|")[1]` looks dangerous, in case `cloudferrylib.scheduler.task.BaseTask.__repr__()` changes, we'll have `IndexError` here. I would suggest simply using `str(task)`.